### PR TITLE
describe how to disable spring boot starter / use sys property widget everywhere

### PR DESCRIPTION
**Pull request by zeitlinger**

### CODE REVIEW

**Comment by jeanbisutti** [comment]
@zeitlinger We have forgotten this config in the doc! Perhaps it could be included in #3758 that contains many configurations? So, we could see the best place to document  how to disable the OpenTelemetry starter in the new doc structure.

**Comment by zeitlinger** [comment]
@jeanbisutti rebase this on top of https://github.com/open-telemetry/opentelemetry.io/pull/3758?

**Review by breedx-splk — APPROVED** [review]
Do we think most devs/users

File: content/en/docs/languages/java/automatic/spring-boot.md [hunk 1]
```diff
@@ -144,6 +144,14 @@ dependencies {
 
 {{% /tab %}} {{< /tabpane>}}
 
+#### Disable data export
+
+You can disable data export, e.g. for testing purposes.
+
+```properties
+otel.sdk.disabled=true
+```
```
Comment (trask) [inline content/en/docs/languages/java/automatic/spring-boot.md:153]: do we have a consistent way in other Java documentation of describing system properties / env vars?

Comment (zeitlinger) [reply content/en/docs/languages/java/automatic/spring-boot.md]: @trask great point - I've used the standard widget and also fixed all the other places

**Comment by trask** [comment]
@breedx-splk I think github swallowed up the second half of your question 😅

**Review by cartermp — APPROVED** [review]
Changes look good to me, @trask what do you think?

**Review by trask — APPROVED** [review]

**Pull request merged**